src/database/seeds/seedAuction.ts

```diff
@@ -15,10 +15,12 @@ export async function seedAuctions() {
     for (let i = 0; i < 10; i++) {
         const title = faker.lorem.words(3)
         const description = faker.lorem.paragraphs(3)
+        const startingPrice = faker.datatype.number(2500)
```

Comment:
This command randomly generate number from 0 to 2500 ?

Reply:
yep